From 554a68039d22093208ce973591370676871feb33 Mon Sep 17 00:00:00 2001
From: "benjitobias@gmail.com" <benjitobias@gmail.com>
Date: Thu, 20 Apr 2017 18:38:50 +0300
Subject: [PATCH] Fixed coding style

---
 04/src/coding_style.c | 20 ++++++++------------
 04/src/hello.c        | 14 ++++----------
 2 files changed, 12 insertions(+), 22 deletions(-)

diff --git a/04/src/coding_style.c b/04/src/coding_style.c
index 9fd8f59..e29d3b1 100644
--- a/04/src/coding_style.c
+++ b/04/src/coding_style.c
@@ -3,35 +3,31 @@
 #include <asm/delay.h>
 #include <linux/slab.h>
 
-int do_work( int * my_int, int retval ) {
+int do_work(int *my_int, int retval)
+{
 	int x;
-	int y=*my_int;
+	int y = *my_int;
 	int z;
-	
-	for(x=0;x< * my_int;++x) {
+
+	for(x = 0; x < *my_int; ++x)
 		udelay(10);
-	}
 
-	if (y < 10 )
-		// That was a long sleep, tell userspace about it
+	if (y < 10)
 		printk("We slept a long time!");
 
 	z = x * y;
-
 	return z;
 }
 
-int
-my_init (void)
+int my_init (void)
 {
 	int x = 10;
 
 	x = do_work(&x, x);
-
 	return x;
 }
 
-void my_exit( void )
+void my_exit(void)
 {
 	return;
 }
diff --git a/04/src/hello.c b/04/src/hello.c
index 157c20a..9ea08ac 100644
--- a/04/src/hello.c
+++ b/04/src/hello.c
@@ -1,6 +1,6 @@
-#include <linux/module.h> // included for all kernel modules
-#include <linux/kernel.h> // included for KERN_DEBUG
-#include <linux/init.h> // included for __init and __exit macros
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/init.h>
 
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("BENJI");
@@ -8,18 +8,12 @@ MODULE_DESCRIPTION("A simple Hello World module");
 
 static int __init hello_init(void)
 {
-    /*
-        Initialize
-    */
     printk(KERN_DEBUG "hello: Hello world!\n");
-    return 0; // Non-zero return means that the module couldn't be loaded.
+    return 0;
 }
 
 static void __exit hello_cleanup(void)
 {
-    /*
-        Unload
-    */
     printk(KERN_DEBUG "hello: Cleaning up module.\n");
 }
 
-- 
2.7.4

